client/src/components/tables/TableAlbumTracks/TableAlbumTracks.js

```diff
@@ -63,17 +72,36 @@ class TableAlbumTracks extends Component {
 
   handleRowDeleteClick = data => {
     this.setState({
-      deleteModal: data,
+      deleteModal: true,
```

Comment:
I think this was changed by accident. The `deleteModal` value should still be `data`